spec/models/cfe/remarks_spec.rb

```diff
@@ -110,7 +110,7 @@ module CFE
           expect(collection.transactions.keys).to match_array expected_transaction_ids
         end
 
-        it "returns a collection with the expected values for a transacton" do
+        it "returns a collection with the expected values for a transactIon" do
```

Comment:
Case correction on `I`

Reply:
LOL - I was only trying to add the `i` in the first place from a previous typo! 🤦‍♂️